metadata/Cadastre/lir_parcels.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Utah LIR Parcels
+
+## ID
+
+cbe4da2a-3edb-4a4a-b082-60f9371c8006
+
+<!--- This is the ID for Parcels, which at the moment is the same for both Basic Parcels and LIR Parcels. May need to be changed later when we have a new ID for LIR Parcels. -->
+
+## Brief Summary
+
+Land Information Records (LIR) Parcel boundaries for all 29 counties in Utah that include a wide variety of attributes beyond the basic information found in Utah Basic Parcels.
+
+## Summary
+
+This dataset contains information such as the county FIPS code, parcel ID, parcel address, generalized land ownership type, currency dates, total market value, tax district, phone number for the county recorder, and the county parcel website if available. Landowners of privately owned parcels are not specified.
```

Comment:
```suggestion
This dataset contains information such as parcel ID, serial number, parcel address, tax exempt type, tax district, land market value, total market value, parcel acres, property class, building square footage, number of floors, and year built.
```

Reply:
If this is a list of every attribute we should remove the `such as` prefix which to implies a subset.

Reply:
great observation and you are correct, it is an incomplete list and highlights what are generally populated and I chose to list out.